packages/data-explorer/src/components/dataView/TableRow.vue

```diff
@@ -28,11 +28,64 @@
         <font-awesome-icon icon="trash"></font-awesome-icon>
       </button>
     </th>
-    <td v-for="(column, index) in visibleColumns" :key="index" class="text-nowrap text-truncate mg-data-column">
-      {{rowData[column.name]}}
+    <td v-for="(column, index) in visibleColumns" :key="index" class="mg-data-column">
+      <DataDisplayCell v-if="typeof getColumnName(column.name) != 'undefined'" :value="getColumnName(column.name)" :type="column.type" :data="column" :rowIndex="rowIndex"></DataDisplayCell>
```

Comment:
when does a column not have a name , and why would the cell be hidden en that case ?

Reply:
column.name is the displayed value (like a date or a string). It's not possible to display a value that is undefined.

Reply:
'column.name is the displayed value'  ? strange